docs/doxygen/pages.dox

```diff
@@ -1,9 +1,9 @@
 /**
 @mainpage
 @anchor ota
-@brief OTA (over-the-air) library.<br><br>
+@brief Over-the-air (OTA) library.<br><br>
```

Comment:
Update the name as "AWS IoT Over-the-air Update Library (Release Candidate)"

Reply:
Addressed in https://github.com/aws/ota-for-aws-iot-embedded-sdk/pull/128/commits/85064419842169800cd88b04dee3c1248cf7c474